cmd/helpMessages.go

```diff
@@ -143,6 +143,20 @@ const resumeJobsCmdShortDescription = "Resume the existing job with the given jo
 const resumeJobsCmdLongDescription = `
 Resume the existing job with the given job ID.`
 
+const removeJobsCmdShortDescription = "Remove all files associated with the given job ID"
+
+const removeJobsCmdLongDescription = `
+Remove all files associated with the given job ID.
+
```

Comment:
Would be real nice to have examples in here too, as we do for most other commands.